app/scripts/utils/test-helpers.js

```diff
@@ -195,3 +195,10 @@ export const mountHGComponent = (prevDiv, prevHgc, viewConf, done, options) => {
 
   return [div, hgc];
 };
+
+export const removeHGComponent = (div) => {
+  // console.log('removing', div);
```

Comment:
This commented out bit probably isn't useful? Unless you expect to continue debugging?

Reply:
removed